docs/android/customisation/settingsMenu.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
```

Comment:
Can the copy be consistent for Android and iOS as far as possible for the description of functionality